codalab/worker/worker_run_state.py

```diff
@@ -357,7 +404,13 @@ def mount_dependency(dependency, shared_file_system):
             self.worker_docker_network.connect(container)
         except docker_utils.DockerUserErrorException as e:
             message = 'Cannot start Docker container: {}'.format(e)
-            logger.warning(message)
+            log_bundle_transition(
+                bundle_uuid=run_state.bundle.uuid,
+                previous_stage=run_state.stage,
+                next_stage=RunStage.CLEANING_UP,
+                reason=message,
```

Comment:
Since the level is `ERROR` it will show the traceback twice: once in message and another time when calling `traceback.format_exc()`. Can we just put `Cannot start Docker Container.` as the reason and keep `message` the same?